homebrew_challenge/src/App.css

```diff
@@ -20,42 +20,7 @@ img#logo {
 .link {
   color: #3c3c3b;
   font-weight: bold;
-  text-decoration: underline;
-}
-
-.sitemap .title {
-  font-weight: bold;
-}
-
-.sitemap .entry:hover,
-.sitemap .entry:active {
-  text-decoration: underline;
-  cursor: pointer;
-}
-
-.sitemap {
-  z-index: 1;
-}
-
-.sitemap .entry {
-  position: relative;
-  width: 20%;
-  left: 40%;
-  z-index: 2;
-}
-
-@media screen and (max-width: 1300px) {
-  .sitemap .entry {
-    width: 40%;
-    left: 30%;
-  }
-}
-
-@media screen and (max-width: 768px) {
-  .sitemap .entry {
-    width: 60%;
-    left: 20%;
-  }
+  text-decoration: none;
```

Comment:
This removes the underline everywhere, including the phase guidance. Is that intentional?

Reply:
yes - modern UX apparently but I think it should go back on the phase guidance as currently not clear that this is a link

Reply:
routemap link is now underlines